scripts/check_frontend_coverage.py

```diff
@@ -26,423 +26,476 @@
-    'suggestion-modal-for-creator-view.controller.ts',
-    'suggestion-modal-for-creator-view.service.ts',
-    'suggestion-modal.service.ts',
-    'SuggestionImprovementTaskObjectFactory.ts',
-    'SuggestionObjectFactory.ts',
-    'suggestions.service.ts',
-    'SuggestionThreadObjectFactory.ts',
-    'summarize-nonnegative-number.filter.ts',
-    'teach-page.controller.ts',
-    'test.extras.ts',
-    'text-input-prediction.service.ts',
-    'text-input-rules.service.ts',
-    'text-input-validation.service.ts',
-    'text-input.tokenizer.ts',
-    'TextInput.ts',
-    'thread-data.service.ts',
-    'thread-status-display.service.ts',
-    'ThreadMessageObjectFactory.ts',
-    'ThreadMessageSummaryObjectFactory.ts',
-    'topic-creation-backend-api.service.ts',
-    'topic-domain.constants.ajs.ts',
-    'topic-domain.constants.ts',
-    'topic-editor-page.constants.ajs.ts',
-    'topic-editor-page.constants.ts',
-    'topic-landing-page.constants.ajs.ts',
-    'topic-landing-page.constants.ts',
-    'topic-landing-page.controller.ts',
-    'topic-rights-backend-api.service.ts',
-    'topic-viewer-backend-api.service.ts',
-    'topic-viewer-domain.constants.ajs.ts',
-    'topic-viewer-domain.constants.ts',
-    'TopicRightsObjectFactory.ts',
-    'topics-and-skills-dashboard-domain.constants.ajs.ts',
-    'topics-and-skills-dashboard-domain.constants.ts',
-    'topics-and-skills-dashboard-page.constants.ajs.ts',
-    'topics-and-skills-dashboard-page.constants.ts',
-    'TopicSummaryObjectFactory.ts',
-    'training-data-editor-panel-modal.controller.ts',
-    'training-data-editor-panel.service.ts',
-    'training-data.service.ts',
-    'training-modal.controller.ts',
-    'training-modal.service.ts',
-    'translation-language.service.ts',
-    'translation-status.service.ts',
-    'translation-tab-active-content-id.service.ts',
-    'translation-tab-active-mode.service.ts',
-    'truncate-at-first-ellipsis.filter.ts',
-    'truncate-at-first-ellipsis.pipe.ts',
-    'truncate-at-first-line.filter.ts',
-    'truncate-at-first-line.pipe.ts',
-    'uiLeafletRequires.ts',
-    'underscores-to-camel-case.filter.ts',
-    'underscores-to-camel-case.pipe.ts',
-    'undo-redo.service.ts',
-    'UnitsObjectFactory.ts',
-    'UpgradedServices.ts',
-    'url.service.ts',
-    'user-email-preferences.service.ts',
-    'user-exploration-permissions.service.ts',
-    'user.service.ts',
-    'UserInfoObjectFactory.ts',
-    'validators.service.ts',
-    'valueGeneratorsRequires.ts',
-    'VoiceoverObjectFactory.ts',
-    'window-dimensions.service.ts',
-    'window-ref.service.ts',
-    'winnowing-preprocessing.service.ts',
-    'WorkedExampleObjectFactory.ts',
-    'wrap-text-with-ellipsis.filter.ts',
-    'wrap-text-with-ellipsis.pipe.ts',
-    'WrittenTranslationObjectFactory.ts',
-    'WrittenTranslationsObjectFactory.ts',
+# NOTE TO DEVELOPERS: do not add any new files to this list without asking
+# @marianazangrossi first.
+BLACKLIST = [
```

Comment:
While this is indeed a blacklist, please still keep its variable name as NOT_FULLY_COVERED_FILENAMES. The term "blacklist" is quite a general term. It's a bit like calling this variable a "list" or a "string". The more specific name that you used before provides more context.

Reply:
Done!